https://issues.redhat.com/browse/ACM-20406--Release note for Cluster reinstallation with SiteConfig (#7968)
open-cluster-management/rhacm-docs: release_notes/acm_whats_new.adoc

* You can now change the name of your local cluster. By default, the hub cluster manages itself as the `local-cluster`. When the `disableHubSelfManagement` field is set to `true`, which disables the local cluster feature, you can change the name. Use 34 or fewer characters for the `<your-local-cluster-name>` value. The `local-cluster` resource and the namespaces reflects the change.

> * You can now reinstall clusters by using the `ClusterInstance` API of the SiteConfig operator. For more information, see link:../mce_acm_integration/siteconfig/siteconfig_cluster_reinstallation.adoc#cluster-reinstallation[Cluster reinstallation with the SiteConfig operator (Technology Preview)].

Reviewer: ```suggestion
* You can now reinstall clusters by reusing the same `ClusterInstance` CR from the SiteConfig operator. For more information, see link:../mce_acm_integration/siteconfig/siteconfig_cluster_reinstallation.adoc#cluster-reinstallation[Cluster reinstallation with the SiteConfig operator (Technology Preview)].
```
